.github/workflows/ci.yaml

```diff
@@ -563,18 +556,31 @@ jobs:
     runs-on: ${{ matrix.os.image }}
 
     steps:
+      - uses: subosito/flutter-action@v2
+        with:
+          channel: "stable"
+      
       - uses: actions/checkout@v2
 
       - uses: dart-lang/setup-dart@v1
 
-      - name: Install dart global dependency
-        run: dart pub global activate ffigen ^6.0.1
-
       # needed by `ffigen`, see https://github.com/dart-lang/ffigen#installing-llvm
       - name: Install llvm dependency (Linux)
         if: ${{ matrix.os.family == 'linux' }}
         run: sudo apt update && sudo apt-get install -y libclang-dev
 
+      - name: Install dart dependencies (single block)
```

Comment:
Hmm why do we need these installations here

Reply:
we still need to install the dependencies, don't we ?

Reply:
I only looked at the code diff at github, and it renders them green, i.e. originally we did not call `pub get` but now we do.

Reply:
reverted in 01ee6157db45b8123bdf1491521221963537939c

Reply:
well... [CI is failing](https://github.com/fzyzcjy/flutter_rust_bridge/runs/7646268043?check_suite_focus=true)... looks like it's required now 😅 
I guess before `dart pub global activate ffigen` would trigger an install.

Reply:
reintroduced in c09c62eeb219068f71d7d118632fb86c7212ec3c

Reply:
I let you resolve this one, just to be sure you acknowledge the changes 👌

Reply:
I see. So no problem :)